packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -62,12 +67,13 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
         {...restProps}
       >
         <div className={styles.icon}>
-          <Icon size="xl" name={getIconFromSeverity(severity)} />
+          <Icon size="xl" name={getIconFromSeverity(severity, useNewErrorIcon)} />
         </div>
 
         <div className={styles.body}>
           <div className={styles.title}>{title}</div>
           {children && <div className={styles.content}>{children}</div>}
+          {additionalContent}
```

Comment:
I haven't used some of the core plugins too much so I haven't seen all the cases of alerts but I can just put it in the same div and in case something is off we'll fix it while testing.